imports/plugins/core/ui-navbar/client/containers/navbar.js

```diff
@@ -19,7 +19,7 @@ function composer(props, onData) {
   }
 
   if (shop && Array.isArray(shop.brandAssets)) {
-    const brandAsset = _.find(this.getShop().brandAssets, (asset) => asset.type === "navbarBrandImage");
+    const brandAsset = _.find(shop.brandAssets, (asset) => asset.type === "navbarBrandImage");
```

Comment:
This is identical to the line in the current `marketplace` branch so I'm confused as to why it's showing this as a change, but it appears to be correct.
https://github.com/reactioncommerce/reaction/blob/marketplace/imports/plugins/core/ui-navbar/client/containers/navbar.js#L22

Reply:
I pulled the latest changes from marketplace , I made a typographical error in my commit message so I reset my HEAD and unknowingly unstaged the change that was made to the `brandAssets`.